ERCS/erc-6932.md

```diff
@@ -0,0 +1,125 @@
+
+The subscription-based [ERC-20](./eip-20.md) token provides a more flexible and convenient way to manage recurring payments. It can be used for a wide range of services and products that require regular payments, such as subscription-based content platforms, gaming services, and more.
+
+The Subscription Token ensures consistency and interoperability across different implementations. Key features include:
+
+- Auto Deduction: Merchants, acting as subscribers, can set the subscription interval and associated payment amount for their services. This information is encoded within the Subscription Token contract, enabling automatic deductions from user balances at regular intervals without requiring manual intervention.
+
+- Balance Check: Users can verify the remaining balance of their subscription tokens at any given time. This transparency empowers users to monitor their subscriptions and make informed decisions regarding their ongoing commitment to the service.
+
+- Flexibility: The Subscription Token framework accommodates various subscription models, such as monthly, quarterly, or annual billing cycles. Additionally, merchants have the option to define trial periods, upgrade/downgrade plans, and cancellation policies, providing a versatile foundation for a wide range of subscription-based businesses.
+
+- Security: The Subscription Token employs established security measures, including the use of cryptographic signatures, to ensure the integrity and authenticity of subscription-related transactions. This protects both users and merchants from unauthorized access and potential malicious activities.
+
+## Specification
+
+Below are the implementaions required by the standard:
+
+
+### `SubscriptionToken`
+
+#### `subscribers`
+
+Returns the list of `addresses` subscribed to the subscription token contract.
+
+#### `subscriptionInfo`
+
+Metadata information of the subscription, like - `subscriptionID`, `subscriptionName`, `subscriptionDesc` and `subscriptionTandC`.
+
+#### `subscriptionFee`
+
+The subscription amount specified that will be deducted in `subscriptionFrequency` interval, when an address subscribes to the subscription token contract.
+
+#### `subscriptionFrequency`
+
+Frequency of subscription, interval at which the `subscriptionFee` will be charged. for example, every 1 day, 1 week or 1 month, denoted in seconds.
+
+#### `subscribe`
+
+Method for subscribing an address to the subscription token contract.
+
+#### `unsubscribe`
+
+Revoke subscription from the subscription token contract, by the subscribed address.
+
+```solidity
```

Comment:
This code block will appear under the `unsubscribe` heading in the Table of Contents. I'd recommend moving it directly under the `SubscriptionToken` heading so it makes more sense.